examples/typescript/template.md

```diff
@@ -0,0 +1,31 @@
+# example.js
+
+```javascript
+_{{example.js}}_
+```
+
+# index.ts
+
+```typescript
+_{{index.ts}}_
+```
+
```

Comment:
webpack.config.js should be shown in README/template too

Reply:
fixup https://github.com/webpack/webpack/pull/9280/commits/c998cc53a6344288c76a2f9b1793e71f9b885eea